### PR TITLE
[RA-1 Ch07] LCM Content

### PR DESCRIPTION
**Pull request by pgoyal01**
1. Move RA-1 4.6 "Logging / Monitoring / Alerting of Control Plane" to RA-1 Ch07. Deleted 4.6 from RA_1 Ch04. Will be added in next commit.
1. Add Preliminary TOC from Issue #1116 in RA-1 Ch 07
1. Add content to RA-1 Ch07

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 1]
```diff
@@ -27,16 +29,74 @@
...
 <a name="7.4"></a>
-## 7.4 Monitoring
-- Metrics exporters, metrics collection
-- Monitoring services
-- Alerting services
--	Logs collection
-
-<a name="7.5"></a>
-## 7.5 Maintenance
+## 7.4 MFVI and VIM Maintenance
```
Comment (ASawwaf) [inline doc/ref_arch/openstack/chapters/chapter07.md]: @pgoyal01 i think NFVI not MFVI

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 2]
```diff
@@ -4,11 +4,13 @@
...
 ## Table of Contents
-* [7.1 Introduction](#7.1)
-* [7.2 Requirements](#7.2)
-* [7.3 NFVI and VIM initial deployment and configuration ](#7.3)
-* [7.4 Monitoring ](#7.4)
-* [7.5 Maintenance ](#7.5)
+* [7.1 Introduction ](#7.1)
+* [7.2 Requirements ](#7.2)
+* [7.3 NFVI and VIM configuration management ](#7.3)
+* [7.4 MFVI and VIM Maintenance ](#7.4)
```
Comment (ASawwaf) [inline doc/ref_arch/openstack/chapters/chapter07.md]: @pgoyal01 i think NFVI not MFVI

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @ASawwaf Thanks -- Corrected

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @ASawwaf Thanks - Corrected

**Comment by pgoyal01** [comment]
Fixes #1116

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 3]
```diff
@@ -27,16 +29,74 @@
...
+While monitoring systems are incredibly useful for active interpretation and investigation, one of the primary
+benefits of a complete monitoring system is letting administrators disengage from the system. ALerts allow the
+specification of situations that make sense to actively manage, while relying on the passive monitoring of the 
+software to watch for changing conditions.
+
+### 7.5.3. Logging,Monitoring, and Analytics (LMA) Framework
+In this section, a possible framework utilizing Prometheus, Elasticsearch and Kibana is given as an example only.
+
+
+<p align="center"><img src="../figures/Figure_4_4_Monitoring_Logging_Framework.png" alt="Monitoring and Logging Framework"></br>
```
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter07.md]: The diagram uses an undefined abbreviation 'PVC'

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 4]
```diff
@@ -27,16 +29,74 @@
...
+Figure 7-1: Monitoring and Logging Framework </p>
+
+The monitoring and logging framework (**Figure 7-1**) leverages Prometheus as the monitoring engine and 
+Fluentd for logging. In addition, the framework uses Elasticsearch to store and organize logs for easy access. 
+Prometheus agents pull information from individual components on every host.  Fluentd, an open source data 
+collector, unifies data collection and consumption for better use and understanding of data. Fluentd captures 
+the access, application and system logs.
```
Comment (iangardner22) [inline doc/ref_arch/openstack/chapters/chapter07.md]: Where do we reference PM - Performance Management?   Do we have a suggested model with collectd and some central collection?

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @iangardner22 Atthis time we only have requirements but we do need a model as you have suggested.

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 5]
```diff
@@ -27,16 +29,74 @@
...
+- Databases maintain a transaction log of all transaction executed including an added, changed and deleted data.
+- Audit logs record chronological documentation of any activities that could have affected a 
+particular operation or event. Data typically includes resources accessed, destination and source 
+addresses, and a timestamp and login information for the person who accessed the resources.
+
+Some of the data is to support the metrics collection specified in the [Reference Model Chapter 4: Infrastructure Capabilities, Metrics and Catalogue](../../ref_model/chapters/chapter04.md).
+
+Logs have multiple operational uses including for:
+
+1. Regulatory Compliance and SIEM featuring the automated gathering, analysis and correlation of log data across all systems and devices across AT&T to provide real-time analysis, event prioritization, reporting, notification and alerting.
```
Comment (kedmison) [inline doc/ref_arch/openstack/chapters/chapter07.md]:
```suggestion
1. Regulatory Compliance and SIEM featuring the automated gathering, analysis and correlation of log data across all systems and devices across an operator to provide real-time analysis, event prioritization, reporting, notification and alerting.
```

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 6]
```diff
@@ -27,16 +29,74 @@
...
+values. Alert definitions are composed of two components: a metrics-based condition or threshold, and an 
+action to perform when the values fall outside of the acceptable conditions.
+
+While monitoring systems are incredibly useful for active interpretation and investigation, one of the primary
+benefits of a complete monitoring system is letting administrators disengage from the system. ALerts allow the
+specification of situations that make sense to actively manage, while relying on the passive monitoring of the 
+software to watch for changing conditions.
+
+### 7.5.3. Logging,Monitoring, and Analytics (LMA) Framework
+In this section, a possible framework utilizing Prometheus, Elasticsearch and Kibana is given as an example only.
```
Comment (kedmison) [inline doc/ref_arch/openstack/chapters/chapter07.md]: Can and should this note be extended to cover fluentd and prometheus (referred to below)?

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 7]
```diff
@@ -27,16 +29,74 @@
...
+and historical data. While values representing the current point in time are useful, it is almost always more 
+helpful to view those numbers in relation to past values to provide context around changes and trends. 
+
+### 7.5.2. Alerting
+Alerting is the responsive component of a monitoring system that performs actions based on changes in metric 
+values. Alert definitions are composed of two components: a metrics-based condition or threshold, and an 
+action to perform when the values fall outside of the acceptable conditions.
+
+While monitoring systems are incredibly useful for active interpretation and investigation, one of the primary
+benefits of a complete monitoring system is letting administrators disengage from the system. ALerts allow the
```
Comment (kedmison) [inline doc/ref_arch/openstack/chapters/chapter07.md]:
```suggestion
benefits of a complete monitoring system is letting administrators disengage from the system. Alerts allow the
```

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 3 above)
Comment (kedmison) [reply doc/ref_arch/openstack/chapters/chapter07.md]: The diagram also refers to Pods, suggesting a particular technology stack is in use.

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 5 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @kedmison Corrected -- Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 7 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @kedmison Corrected -- Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 3 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @iangardner22 @kedmison Deleted original Figure $-4 and replaced with updated Figure 7-1. Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 6 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @kedmison Corrected - Thanks

**Comment by pgoyal01** [comment]
@karinesevilla Done - Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 8]
```diff
@@ -27,16 +29,74 @@
...
 -	Solution evolution by adding new OpenStack services or additional components
 
+<a name="7.5"></a>
+## 7.5 Logging, Monitoring and Analytics
+- Logging
+- Monitoring 
+- Alerting
+- Logging, Monitoring, and Analytics (LMA) Framework
+
+### 7.5.1. Logging
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter07.md]: Please add these subheadings in ToC

File: doc/ref_arch/openstack/chapters/chapter07.md [hunk 9]
```diff
@@ -27,16 +29,74 @@
...
+While monitoring systems are incredibly useful for active interpretation and investigation, one of the primary
+benefits of a complete monitoring system is letting administrators disengage from the system. Alerts allow the
+specification of situations that make sense to actively manage, while relying on the passive monitoring of the 
+software to watch for changing conditions.
+
+### 7.5.3. Logging,Monitoring, and Analytics (LMA) Framework
+In this section, a possible framework utilizing Prometheus, Fluentd, Elasticsearch and Kibana is given as an example only.
+
+
+<p align="center"><img src="../figures/Figure_7_1_Monitoring_Logging_Framework.png" alt="Monitoring and Logging Framework"><b>
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter07.md]: The image has a typo for word `visualisation`

Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @deebhatia Corrected - Thanks

File: doc/ref_arch/openstack/chapters/chapter07.md (hunk 8 above)
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter07.md]: @deebhatia Done - Thanks

**Pull request merged**